src/os/bluestore/BlueStore.cc

```diff
@@ -17521,9 +17521,6 @@ int BlueStore::_do_write_v2(
   o->extent_map.fault_range(db, offset, length);
```

Comment:
I think the PR fix is ok, but not what troubles me is that we do some "around-searching" for blobs and extents nearby in _maybe_reuse_blob etc.
But we do `fault_range` on  range offset~length.
I guess we need to make sure that one of conditions hold:
a) we never produce mapping that goes outside the range offset~length
or
b) we expand `fault_range` so much that we will not look outside
I think b) is better as there was hard limit of lookaround = min_alloc_size.